sccm/osd/deploy-use/create-a-task-sequence-to-upgrade-an-operating-system.md

```diff
@@ -144,6 +144,15 @@ Starting in version 1802, the default task sequence template for Windows 10 in-p
 
 - **Network/wired connection checks**: Add steps in this group to check whether the computer is connected to a network, and isn't using a wireless connection. This action requires a custom script or utility to perform this check.  
 
+- **USB drive checks**: Add steps in this group to check whether a USB drive is connected to the computer. This action requires a custom script or utility. For example: Using WbemTest, connect to the `root\cimv2` namespace. Then run the following query: `Select * From Win32_Diskdrive where InterfaceType = "USB"`. If it returns any results, a USB drive is connected to the computer. Otherwise, no USB drive is connectd.  
+
+- **Device OS version, model, and manufacturer information**: Add steps in this group to check the OS version, model, and manufacturer of the computer. This action requires a custom script or utility to perform this check. For example: Using WbemTest, connect to the `root\cimv2` namespace. Then run the following queries: `Select model, manufacturer from Win32_Computersystem` and `Select Version From Win32_OperatingSystem`. 

```

Comment:
Make and model are variables that also come with the [Set Dynamic Variables](https://docs.microsoft.com/sccm/osd/understand/task-sequence-steps#BKMK_SetDynamicVariables) step. So they can use a custom script or utility, but it's far easier to just add that step. 

I don't think we have any TS variable for OS version, but we should double-check with engineering. But I know you can use conditions based on it. So want to know more about why we should add OS version here?